test/src/state.js

```diff
@@ -51,6 +54,12 @@ function processArgs(state, argv) {
       case "--count":
         state.count = +argv[++i];
         break;
+      case "--driverPath":
+        state.driverPath = +argv[++i];
```

Comment:
`state.driverPath` is not a number, so you need to remove the `+` before `argv[++i]`

Reply:
🤦